Automattic/wp-calypso
# Update the paid media flow to match the GF spec.

Related to Automattic/growth-foundations#25

## Proposed Changes

This PR updates the existing paid media flow on the classic sign-up framework per the spec of the GF signup initiative. i.e. it acts the same as the current main onboarding flow, except that the term toggle is changed to the 1y/2y one. It's also a good timing to remove the copied version of the deprecated plans grid, which in turn fixes #77653 .

As announced in p1689155415551519/1689088443.321509-slack-C059R8S4ALT, the deadline for us to build the Stepper version of it has passed and we still have pending issues to fix found by @skim1220 in Automattic/growth-foundations#72#issuecomment-1638802127 . It's a strong signal that we should stop the loss, unblocking the GF signup work by going back to the classic signup flow, and forming a separate project for migrating the main onboarding flow onto Stepper from what we've learned.

## Testing Instructions

Go through /start/onboarding-pm and check if it works as `/start/` in general. Two things especially worths checking:

* The term toggle in the plans step should be the 1y/2y one.
* Picking a paid domain and then choosing the Free plan in the plans step, the paid domain + Free plan modal should appear.



## Pre-merge Checklist
